stac_model/torch/export.py

```diff
@@ -0,0 +1,137 @@
+import logging
+import tempfile
+from collections.abc import Sequence
+
+import torch
+import yaml
+from torch.export.dynamic_shapes import Dim
+from torch.export.pt2_archive._package import package_pt2
+
+from .utils import aoti_compile_and_extract, create_example_input_from_shape, extract_module_arg_names
+
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
The level should not be enforced here, but instead be applied by whichever script/test where this information is needed to avoid polluting the output of every user of this module/utilities.